Makefile

```diff
@@ -232,38 +232,24 @@ protogen_show: ## A simple `find` command that shows you the generated protobufs
 protogen_clean: ## Remove all the generated protobufs.
 	find . -name "*.pb.go" | grep -v -e "prototype" -e "vendor" | xargs -r rm
 
-# IMPROVE: Look into using buf in the future; https://github.com/bufbuild/buf.
-PROTOC = protoc --experimental_allow_proto3_optional --go_opt=paths=source_relative
-PROTOC_SHARED = $(PROTOC) -I=./shared
-
 .PHONY: protogen_local
 protogen_local: go_protoc-go-inject-tag ## Generate go structures for all of the protobufs
-	# Shared
-	$(PROTOC) -I=./shared/core/types/proto --go_out=./shared/core/types ./shared/core/types/proto/*.proto
-	$(PROTOC) -I=./shared/messaging/proto  --go_out=./shared/messaging  ./shared/messaging/proto/*.proto
-	$(PROTOC) -I=./shared/codec/proto      --go_out=./shared/codec      ./shared/codec/proto/*.proto
-
-	# Runtime
-	$(PROTOC_SHARED) -I=./runtime/configs/proto  --go_out=./runtime/configs ./runtime/configs/proto/*.proto
-	$(PROTOC_SHARED) -I=./runtime/genesis/proto  --go_out=./runtime/genesis ./runtime/genesis/proto/*.proto
-	protoc-go-inject-tag -input="./runtime/genesis/*.pb.go"
-
-	# Persistence
-	$(PROTOC_SHARED) -I=./persistence/indexer/proto 	--go_out=./persistence/indexer ./persistence/indexer/proto/*.proto
-	$(PROTOC_SHARED) -I=./persistence/proto         	--go_out=./persistence/types   ./persistence/proto/*.proto
+# TODO: Organize this code with a basic for loop
```

Comment:
@gokutheengineer I opened the PR and am seeing that we reverted some changes I introduced over the past couple of days. Can you merge with master and make sure it's good to review before I take a deeper look?

Reply:
checked out the Makefile from main branch to this branch, it is fixed now.